doc/source/serve/deployment.rst

```diff
@@ -206,6 +206,122 @@ Please refer to the Kubernetes documentation for more information.
 Monitoring
 ==========
 
+Logging
+-------
+
+Logging in Ray Serve is simple and uses Python's standard logging facility.
+
+.. note::
+
+  For an general overview of logging in Ray, see `Ray Logging <../ray-logging.html>`__.
+
+Tracing Backends and Replicas
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When looking through log files of your Ray Serve application, it is useful to know which backend and replica each log line originated from.
+To automatically include the current backend tag and replica tag in your logs, simply call
+``logger = logging.getLogger("ray")``, and use ``logger`` within your backend code:
+
+.. literalinclude:: ../../../python/ray/serve/examples/doc/snippet_logger.py
+  :lines: 1, 9, 11-13, 15-16
+
+Querying a Serve endpoint with the above backend will produce a log line like the following:
+
+.. code-block:: bash
+
+  INFO -- Some info! component=serve backend=my_backend replica=my_backend#krcwoa
```

Comment:
can you update this? it should print ray logging style now

Reply:
Thanks, good catch.  Updated